device/inventec/x86_64-inventec_d7054q28b-r0/plugins/sfputil.py

```diff
@@ -207,4 +207,4 @@ def reset(self, port_num):
         reg_file.close()
 
         return True
-
+    
```

Comment:
Instead of removing the extra trailing newline, it appears you added spaces? Please remove this line altogether.